src/Spectre.libWavelet.Tests/ConvolutionTest.cpp

```diff
@@ -31,16 +31,25 @@ namespace
 
     class ConvolutionTest : public ::testing::Test
     {
+    public:
+        void InitializeSignal(Signal& signal)
```

Comment:
Can't we just go for producing this signal inside and returning it?

On the other hand, isn't it just an equivalent of calling `spectre::core::functional::range<DataType>(signal.size())`?